generators/form/templates/formComplex.js.ejs

```diff
@@ -15,19 +85,131 @@ const formConfig = {
   },
   title: '<%= appName %>',
   defaultDefinitions: {
+    fullName,
+    ssn,
+    date,
+    dateRange,
+    usaPhone
   },
   chapters: {
-    chapter1: {
-      title: 'Chapter 1',
+    chapterApplicantInfo: {
+      title: 'Applicant Information',
+      pages: {
+        [formPages.applicantInfo]: {
+          path: 'applicant-information',
+          title: 'Applicant Information',
+          uiSchema: {
+            [formFields.fullName]: fullNameUI,
+            [formFields.ssn]: ssnUI
+          },
+          schema: {
+            type: 'object',
+            required: [formFields.fullName],
+            properties: {
+              [formFields.fullName]: veteranFullName,
+              [formFields.ssn]: veteranSocialSecurityNumber,
+            }
+          }
+        }
+      }
+    },
+    chapterServiceHistory: {
+      title: 'Service History',
+      pages: {
+        [formPages.serviceHistory]: {
+          path: 'service-history',
+          title: 'Service History',
+          uiSchema: {
+            [formFields.toursOfDuty]: toursOfDutyUI
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              [formFields.toursOfDuty]: toursOfDuty
+            }
+          }
+        }
+      }
+    },
+    chapterAdditionalInfo: {
+      title: 'Additional Information',
       pages: {
-        page1: {
-          path: 'first-page',
-          title: 'First Page',
+        [formPages.contactInfo]: {
+          path: 'contact-information',
+          title: 'Contact Information',
+          uiSchema: {
+            [formFields.address]: _.set(
+              'ui:validations[1]',
+              validateCentralMailPostalCode,
```

Comment:
We will likely want to demonstrate a simple usage of the common address uiSchema here to start (example from [686 form](https://github.com/department-of-veterans-affairs/vets-website/blob/d932c077011b72d0b5bf07423bd6eb576b22fd36/src/applications/disability-benefits/686/config/form.js#L187-L205)):
```
        claimantInformation: {
          path: 'claimant-information',
          title: 'Applicant Information',
          depends: (formData) => !isVeteran(formData),
          uiSchema: {
            claimantSocialSecurityNumber: _.merge(ssnUI, {
              'ui:title': 'Your Social Security number'
            }),
            claimantAddress: address.uiSchema('Your address'),
          },
          schema: {
            type: 'object',
            required: ['claimantSocialSecurityNumber'],
            properties: {
              claimantSocialSecurityNumber,
              claimantAddress: address.schema(fullSchema686, true),
            }
          }
        }
```

Reply:
Okay, I'll try to work that in.